source/Messages.h

```diff
@@ -33,10 +33,15 @@ class Messages {
 		int step;
 		std::string message;
 	};
+    
+    enum class Importance {
```

Comment:
Let's request a smaller type, e.g. `int_least8_t`.

Reply:
Great suggestion. I made it uint_least8_t.